src/main/java/io/kestra/plugin/fs/local/AbstractLocalTask.java

```diff
@@ -21,14 +19,22 @@
 @Getter
 @NoArgsConstructor
 public abstract class AbstractLocalTask extends Task {
+    static final String ALLOWED_PATHS = "allowedPaths";
 
-    @Schema(
-        title = "Allowed paths for local filesystem access",
-        description = "List of allowed paths for local filesystem access." +
-            " Tasks will only be able to operate on files within these paths."
-    )
-    @NotNull
-    protected Property<List<String>> allowedPaths;
+    protected List<String> getAllowedPaths(RunContext runContext) {
```

Comment:
```suggestion
    protected List<String> allowedPaths(RunContext runContext) {
```

If you use the get prefix, this will be seen as a getter by our serialization layer and be displayed as an attribute that you can set which is not the case.

Reply:
Or maye not as there is a parameter :thinking:
But better to be safe and change the name of the function.